src/components/Compare/columns.tsx

```diff
@@ -163,29 +164,58 @@ class VaccinationsColumn extends MetricColumn {
   }
 }
 
+class HospitalizationsDensityColumn implements ColumnDefinition {
+  columnId = HOSPITALIZATIONS_DENSITY_ID;
+
+  name = 'Hospitalizations per 100k';
+
+  desiredWidthPercent = 14;
+  // Make it wide enough so progress bar doesn't look squashed on mobile.
+  minWidthPx = 130;
```

Comment:
This was specifically so the vaccine progress bar would show up nicely. We probably don't need it for this column?

Reply:
ah, looks like the comment is unrelated (oops) but we still need minWidthPx to make the header look right